src/pages/fr/guides/deploy/buddy.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Déployez votre site Astro avec Buddy
+description: Comment déployer votre site Astro sur le web en utilisant Buddy.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+Vous pouvez déployer votre projet Astro en utilisant [Buddy](https://buddy.works/), une solution CI/CD qui peut construire votre site et le pousser vers de nombreuses cibles de déploiement différentes, y compris les serveurs FTP et les fournisseurs d'hébergement en nuage.
```

Comment:
`hébergement en nuage.`

Seems not a correct translation, "cloud" is not better like `hébergement sur un cloud` ?